search-enterprise/latest/using/index.md

```diff
@@ -232,23 +232,24 @@ In the REST API you can specify the timezone to be used in search for date range
 
 ### Search using date math
 
-Date range queries can be even more powerful when applying date math functions. AFTS supports adding and subtracting periods, as well as rounding:
+Date range queries can be more powerful when applying date math functions. AFTS supports adding and subtracting periods, as well as rounding:
 
-| AFTS query | Description | Notes |
-| ---------- | ----------- | ----- |
-| `acme:projectStartDate:[NOW TO NOW+1DAY>` | Documents with a project start date in the next twenty four hours | |
-| `acme:projectStartDate:[NOW/DAY TO NOW/DAY+1DAY>` | Documents with a project start date today (between midnight and midnight UTC) | Note the subtle difference with the previous query |
-| `acme:projectStartDate:[NOW-1MONTH/YEAR TO NOW-1MONTH/YEAR+1DAY>` | Documents with a project start date in the first day of the year that last month fell in | It's possible to chain date math functions together |
-| `cm:created:[2020-11-01T12:34:00/YEAR TO NOW>` | Documents created since the start of 2020. | It's also possible to apply date math to absolute points in time. |
+| AFTS query | Description |
+| ---------- | ----------- |
+| `acme:projectStartDate:[NOW TO NOW+1DAY>` | Documents that have a project start date in the next twenty four hours. |
+| `acme:projectStartDate:[NOW/DAY TO NOW/DAY+1DAY>` | Documents that have a project start date today and between midnight and midnight (UTC), **Note:** The minor difference between this query and the one above. |
+| `acme:projectStartDate:[NOW-1MONTH/YEAR TO NOW-1MONTH/YEAR+1DAY>` | Documents with a project start date in the first day of the year that last month fell in
+documents with a project start date in (a day). **Note:** It's possible to chain date math functions at the same time. |
```

Comment:
I don't understand this change - you seem to have added half a sentence? ("documents with a project start date in (a day)")

Reply:
I'm not sure the note makes sense either.

Reply:
Hi Tom, 

Thanks. I made the change based on what we spoke about on Slack. 
Can you provide with a clear sentence of what you need? 
The note is from the original change and provided to me, if you're not happy with it let me know something that is more suitable.
Cheers

Reply:
I prefer the note that I wrote: "It's possible to chain date math functions together"

The first sentence no longer makes sense at the moment - I'm not sure it's even a single sentence:
> Documents with a project start date in the first day of the year that last month fell in
> documents with a project start date in (a day).

I originally wrote:
> Documents with a project start date in the first day of the year that last month fell in

but you didn't find this easy to read.  Perhaps this is clearer?
> Documents with a project start date in the first day of this year, or in the first day of last year if it is currently January.